src/EmpiricalModeDecomposition.jl

```diff
@@ -98,56 +98,57 @@ end
 Intermediate results of the CEEMD Iteration
 """
 struct CEEMDState
-  yvec
-  iter_ens
-  imf_state_ens
-  finished::Bool
+    yvec
+    iter_ens
+    imf_state_ens
+    finished::Bool
 end
 Base.IteratorSize(::Type{CEEMDIterable{U,V,T}}) where {U,V,T} = Base.SizeUnknown()
 
 function Base.iterate(iter::CEEMDIterable)
-  imf0 = mean([sift(iter.yvec+noise,iter.xvec,0.1) for noise in iter.noise_ens])
-  iter_ens = EMDIterable.(iter.noise_ens,[iter.xvec])
-  state_ens = Tuple[]
-  imf_state_ens = iterate.(iter_ens)
-  imf0,CEEMDState(iter.yvec-imf0,iter_ens,imf_state_ens,false)
+    imf0 = mean([sift(iter.yvec+noise,iter.xvec,0.1) for noise in iter.noise_ens])
+    iter_ens = EMDIterable.(iter.noise_ens, [iter.xvec])
+    state_ens = Tuple[]
+    imf_state_ens = iterate.(iter_ens)
+    imf0, CEEMDState(iter.yvec-imf0, iter_ens, imf_state_ens, false)
 end
 
-function Base.iterate(iter::CEEMDIterable,state::CEEMDState)
+function Base.iterate(iter::CEEMDIterable, state::CEEMDState)
 
-  vstop = var(iter.yvec)*1e-10
+    vstop = var(iter.yvec)*1e-10
 
-  if state.finished
+    if state.finished
 
-    return nothing
+        return nothing
 
-  elseif sum(abs,state.yvec)>vstop && !ismonotonic(state.yvec)
+    elseif sum(abs,state.yvec)>vstop && !ismonotonic(state.yvec)
 
-      imf = vec(median(hcat([sift(state.yvec+noise[1],iter.xvec,0.1) for noise in state.imf_state_ens]...),dims = 2))
+        imf = vec(median(hcat([sift(state.yvec+noise[1], iter.xvec, 0.1) for
```

Comment:
I like that you try to bring all lines in the 92 character limit, but I think here it is rather confusing to split it in the middle of the inner vector. Could you please revert this line and add a `# TODO: Format for clarity`?
I am going to think about a better way to write this line in the next days.

Reply:
Sure